### DISCUSSION TITLE
Play every two hour

### DISCUSSION THREAD

**Discussion by UseRadio** [savonet/liquidsoap#1752] [Q&A]
Hello,

How can I  schedule a playlist to be played every two hour or three hour?
Thanks in advance

**Comment by toots** [comment]
Hi! I've converted this issue into a discussion. Let's continue chatting here!

**Comment by smimram** [comment]
You could try something like this
```
default = ...
p = playlist(reload=7200, loop=false, "my_playlist.pls")
s = fallback([p, default])
output(s)
```
Here `p` is the playlist, which is reloaded every 7200 seconds (= 2h). After it has been played, it will not be available until the next reload and we we fallback on `default`.
